weblab/datasets/forms.py

```diff
@@ -50,3 +51,26 @@ class DatasetFileUploadForm(forms.ModelForm):
     class Meta:
         model = DatasetFile
         fields = ['upload']
+
+
+class DatasetTransferForm(UserKwargModelFormMixin, forms.Form):
```

Comment:
Yes I believe it can but I'm not sure of the correct syntax. Should I be referencing the `EntityTransferForm` in` dataset\views.py ` or do I need a sub form in `dataset\forms.py`. I can only find examples of sub forms for model forms.

Reply:
I think in `datasets/views.py` you can use `from entities.forms import EntityTransferForm` and then use that where currently you have `DatasetTransferForm`.

Reply:
Though given that a dataset is not an entity, perhaps it makes sense to move that class into `accounts/forms.py` and name it `OwnershipTransferForm`?

Reply:
Oh, and it doesn't need to inherit from `UserKwargModelFormMixin` I think, as you don't make use of the logged in user info, only the manually entered email address.

Reply:
interestingly when I remove `UserKwargModelFormMixin` I get the following error `__init__() got an unexpected keyword argument 'user'`

Reply:
Ah, you also need to remove the corresponding base class from the view(s) which provides that argument. See https://django-braces.readthedocs.io/en/latest/form.html#userformkwargsmixin for how this works.